rotkehlchen/tests/unit/decoders/test_yearn.py

```diff
@@ -894,3 +919,128 @@ def test_yearn_v3_curve_savings_withdraw(
         counterparty=CPT_CURVE,
         address=curve_savings_vault.evm_address,
     )]
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('ethereum_accounts', [['0xFa4Ebcb83902Bb1106b85Bb3D4916Dfd72E06721']])
+def test_yearn_staking_withdraw(
+        ethereum_inquirer: 'EthereumInquirer',
+        ethereum_accounts: list['ChecksumEvmAddress'],
+        yearn_yfi_eth_gauge: 'EvmToken',
+) -> None:
+    tx_hash = deserialize_evm_tx_hash('0x1420b1caf7ee2d8d59f2814d2254ac9f72eb8d2d64fdacc1a4b393184c75d841')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(evm_inquirer=ethereum_inquirer, tx_hash=tx_hash)
+    assert events == [EvmEvent(
+        tx_ref=tx_hash,
+        sequence_index=0,
+        timestamp=(timestamp := TimestampMS(1760597219000)),
+        location=Location.ETHEREUM,
+        event_type=HistoryEventType.SPEND,
+        event_subtype=HistoryEventSubType.FEE,
+        asset=A_ETH,
+        amount=(gas_amount := FVal('0.000031728668')),
+        location_label=(user_address := ethereum_accounts[0]),
+        notes=f'Burn {gas_amount} ETH for gas',
+        counterparty=CPT_GAS,
+    ), EvmEvent(
+        tx_ref=tx_hash,
+        sequence_index=1,
+        timestamp=timestamp,
+        location=Location.ETHEREUM,
+        event_type=HistoryEventType.SPEND,
+        event_subtype=HistoryEventSubType.RETURN_WRAPPED,
+        asset=yearn_yfi_eth_gauge,
+        amount=(spend_amount := FVal('3.256705603870174511')),
+        location_label=user_address,
+        notes=f'Return {spend_amount} yG-yvCurve-YFIETH to a Yearn Staking gauge',
+        counterparty=CPT_YEARN_STAKING,
+        address=ZERO_ADDRESS,
+    ), EvmEvent(
+        tx_ref=tx_hash,
+        sequence_index=2,
+        timestamp=timestamp,
+        location=Location.ETHEREUM,
+        event_type=HistoryEventType.WITHDRAWAL,
+        event_subtype=HistoryEventSubType.REDEEM_WRAPPED,
+        asset=Asset('eip155:1/erc20:0x790a60024bC3aea28385b60480f15a0771f26D09'),
+        amount=spend_amount,
+        location_label=user_address,
+        notes=f'Withdraw {spend_amount} yvCurve-YFIETH from Yearn Staking gauge yGauge Curve YFI-ETH Pool yVault',  # noqa: E501
+        counterparty=CPT_YEARN_STAKING,
+        address=yearn_yfi_eth_gauge.evm_address,
+    ), EvmEvent(
+        tx_ref=tx_hash,
+        sequence_index=3,
+        timestamp=timestamp,
+        location=Location.ETHEREUM,
+        event_type=HistoryEventType.RECEIVE,
+        event_subtype=HistoryEventSubType.REWARD,
+        asset=Asset('eip155:1/erc20:0x41252E8691e964f7DE35156B68493bAb6797a275'),
+        amount=(reward_amount := FVal('0.622517149171265675')),
+        location_label=user_address,
+        notes=f'Claim {reward_amount} dYFI from Yearn Staking gauge yGauge Curve YFI-ETH Pool yVault',  # noqa: E501
```

Comment:
the token name is sufficient here. 'Yearn Staking gauge' is redundant since 'gauge' is already part of the token name, and the counterparty makes the context clear.